Modix.Data.Test/Assertions/DbContextSequenceExtensions.cs

```diff
@@ -22,7 +22,7 @@ public static class DbContextSequenceExtensions
 
             var table = tableSelector.Compile().Invoke(context).Local;
 
-            valueGenerator.SetValue(default(TProperty));
+            valueGenerator.SetValue(default(TProperty)!);
```

Comment:
A `notnull` constraint wouldn't change anything. We'd still be creating a `default(TProperty)`, which is `null` when `TProperty` is a reference type.

Reply:
`TProperty` can never be a reference type for a value generator. I mean, syntactically it can, but, like I said above...

So, a constraint of `TProperty : struct` should work, then?

Feel free to just leave it like this if you want, this whole module needs gutted eventually. The problem solves is no longer an issue in EF Core 3.